### PR TITLE
android onLowMemory

### PR DESCRIPTION
**Pull request by MatejMagat305**
### Description:
This is only 2 lines of code - this is trivial change, clean memory at Android OS request ...

### CODE REVIEW

**Review by changkun — APPROVED** [review]
Looks like a reasonable change to me. (why it was not there before? the API was introduced in go1.1…)

File: internal/driver/mobile/app/android.go [hunk 1]
```diff
@@ -268,6 +269,7 @@ func onConfigurationChanged(activity *C.ANativeActivity) {
 
 //export onLowMemory
 func onLowMemory(activity *C.ANativeActivity) {
+	debug.FreeOSMemory()
```
Comment (changkun) [inline internal/driver/mobile/app/android.go:272]: I just recall that we could also call runtime.GC() before manually free OS memories so this is more effective:

```suggestion
	runtime.GC()
	debug.FreeOSMemory()
```

File: internal/driver/mobile/app/android.go [hunk 2]
```diff
@@ -57,6 +57,7 @@ import (
 	"log"
 	"mime"
 	"os"
+	"runtime/debug"
```
Comment (changkun) [inline internal/driver/mobile/app/android.go:60]:
```suggestion
	"runtime"
	"runtime/debug"
```

**Review by changkun — CHANGES_REQUESTED** [review]

**Comment by MatejMagat305** [comment]
@changkun ok you aare bigger expert than me - I was thinking the freeOSMemory call gc exciplit ...

**Review by changkun — APPROVED** [review]
Thanks! There are two types of collectors in the runtime, one is the normal GC, and the other is called scavenger. I think debug.FreeOSMemory is only effective to the scavenger, which returns runtime managed memory to OS.

**Pull request merged**